static/js/services/screensharing.js

```diff
@@ -43,6 +46,20 @@ define(['underscore', 'text!partials/screensharedialogff.html', 'webrtc.adapter'
 			chromeExtension.e.on("available", _.bind(function() {
 				this.initialize();
 			}, this));
+			firefoxExtension.e.on("available", _.bind(function() {
+				this.initialize();
+			}, this));
+		};
+
+		Screensharing.prototype.globalNotify = function() {
```

Comment:
I need these to notify the plugin when to insert the domain and when to remove it from Firefox.